src/webgpu/util/copy_to_texture.ts

```diff
@@ -104,17 +139,44 @@ export class CopyToTextureUtils extends GPUTest {
           A: orientedPixels[pixelPos * 4 + 3] / divide,
         };
 
-        if (!srcPremultiplied && dstPremultiplied) {
+        if (requireUnpremultiplyAlpha && rgba.A !== 0.0) {
```

Comment:
Rather than silently skipping when A=0, should probably assert that R=G=B=0 as well.

(Normally I would say we should assert R<=A && G<=A && B<=A, but it occurs to me that would be valid if we ever have extended-gamut premultiplied values.)

Reply:
Done.